packages/auth/src/server.ts

```diff
@@ -100,7 +100,7 @@ export const {
           ...session.user,
           id: user.id,
           role: userRoles,
-          createAt: user.createdAt,
+          createdAt: user.createdAt,
```

Comment:
idk if this was intentionally spelt `createAt`?

Reply:
woops